.github/workflows/release-libs.yml

```diff
@@ -1,4 +1,4 @@
-name: Release Platform Libraries
```

Comment:
There are workflow triggers that use the name in the `workflow_run` trigger that would break with this rename

Reply:
Let me undo this one, it's more for the language SDK bindings.